server/admin_management/api/app/repositories/user_repository.py

```diff
@@ -36,12 +44,26 @@ def get_user_by_cognito_user_id(self, cognito_user_id: str) -> models.FamUser:
             .one_or_none()
         )
 
-    def get_users(self) -> List[models.FamUser]:
-        return self.db.query(models.FamUser).all()
+    # --- Create ---
 
     def create_user(self, fam_user: schemas.FamUserDto) -> models.FamUser:
         user_dict = fam_user.model_dump()
         db_item = models.FamUser(**user_dict)
         self.db.add(db_item)
         self.db.flush()
         return db_item
+
+    # --- Update ---
+
+    def update(self, user_id, update_values: dict, requester: str) -> int:
+        LOGGER.debug(
+            f"Update on FamUser {user_id} with values: {update_values} " +
+            f"for requester {requester}"
+        )
+        update_count = (
+            self.db.query(models.FamUser)
+            .filter(models.FamUser.user_id == user_id)
+            .update({**update_values, models.FamUser.update_user: requester})
```

Comment:
When record is being updated, also records "update_user" as who updates this record.